app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
+  <div class="col-md-3"> </div>
+    <div class="col-md-5 bound-back">
+    <div class="row center-block">
+      <h2><%= t(".login") %></h2>
+      <img src="/assets/logo.png" class="img-circle img-logo-small">
+    </div>
+      <%= form_for(:session, url: login_path) do |f| %>
+      <div class="input-group">
+        <span class="input-group-addon"><span class="glyphicon glyphicon-user"></span></span>
+        <%= f.email_field :email, class: "form-control", placeholder: t(".email") %>
+      </div>
+      <div class="input-group">
+        <span class="input-group-addon"><span class="glyphicon glyphicon-lock"></span></span>
+        <%= f.password_field :password, class: "form-control", placeholder: t(".password") %>
+      </div>
+      <center>
+        <%= f.check_box :remember_me %>
+        <small><%= t(".remem_pass") %></small>

```

Comment:
Same here

Reply:
How to know where it is needed remove optional () ?
ex:<%= f.submit t(".login"), class: "btn-success btn-login" %>
Why I was remove optional () in here, it not working.
Pls help me ...